providers/SupportedAssetsProvider.tsx

```diff
@@ -0,0 +1,62 @@
+import {
```

Comment:
Don't think you would need this, so let's remove it

Reply:
I thought this one might be useful for cennzx, what do you reckon?

Reply:
yes definitely, was just about to write up logic to get cennz assets. Once this is in I can pair them up with their assets logos too

Reply:
true, yip agree! note that I'm updating it atm on Litho so it can fetch during build time (via `getStaticProps`), will make a PR here once that done